### DISCUSSION TITLE
Latency and high buffer settings

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#459] [General]
When reading the forums I read about people disappointed about the latency audiogridder adds. When I mix I always use higher buffer settings and I would not use audiogridder for tracking live vocals or instruments.

Would I have any additional issues with latency based on my workflow

**Comment by apohl79** [comment]
Probably not. It really comes down to your network performance.

**Comment by CaBuffalo** [comment]
I'm wondering if you have any problem using AG.

I think what you have been doing is right.

On Tue, Mar 30, 2021 at 11:48 PM bsjdsp ***@***.***> wrote:

> When reading the forums I read about people disappointed about the latency
> audiogridder adds. When I mix I always use higher buffer settings and I
> would not use audiogridder for tracking live vocals or instruments.
>
> Would I have any additional issues with latency based on my workflow
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/459>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EK4ETMAXYMCYT5KNLTGLAUPANCNFSM42D3SJRQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Clintcinnati** [comment]
Something I have noticed is that if i have wifi activated on the server, and that server is also connected via ethernet (or other wired direct connection) there are times where the plugins will try and often successfully connect to the server via wifi rather than the faster wired connection, causing horrid performance, so I make very sure wifi is turned off on the server machine before spawning the sever.

**Comment by CaBuffalo** [comment]
This is the network connection that causes the performance problem. if Wifi
works in a certain way, that means AG works fine except the slow connection
causes the drop offs and CPU spikes.

On Sun, Apr 4, 2021 at 12:49 PM Clintcinnati ***@***.***>
wrote:

> Something I have noticed is that if i have wifi activated on the server,
> and that server is also connected via ethernet (or other wired direct
> connection) there are times where the plugins will try and often
> successfully connect to the server via wifi rather than the faster wired
> connection, causing horrid performance, so I make very sure wifi is turned
> off on the server machine before spawning the sever.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/459#discussioncomment-568539>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BIVGUFM3OMXZBQC63THC7GJANCNFSM42D3SJRQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Clintcinnati** [reply]
I understand it to be generally bad practice to have wifi on at all when using a DAW, and I connect the audiogridder server directly to the DAW computer, with no hub between- one wire direct computer to computer (IP over Thunderbolt).
